### PR TITLE
AAP-4381 New content on LDAP configuration for private automation hub

### PR DESCRIPTION
**Pull request by jself-sudoku**
Added new reference module w/ two sets of instructions (discrete headings): 

-  LDAP configuration on private automation hub (`ref-ldap-config-on-pah.adoc`)
    - Setting up your inventory file variables
    - Configuring extra LDAP parameters

- Updated `assembly-platform-install-scenario.adoc` to include `xref` and the `include::` for `ref-ldap-config-on-pah.adoc`.

### CODE REVIEW

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+You must set the following six variables in your {PlatformName} installer inventory file to configure your {PrivateHubName} for LDAP authentication:
+
+* `automationhub_authentication_backend`
+* `automationhub_ldap_server_uri`
+* `automationhub_ldap_bind_dn`
+* `automationhub_ldap_bind_password`
+* `automationhub_ldap_user_search_base_dn`
+* `automationhub_ldap_group_search_base_dn`
+
+If any of these variables are missing, the Ansible Automation  installer will  not complete the installation.
```
Comment (ianf77) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:15]: looks like two spaces between Automation and Installer. I might be wrong.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+ldap_extra_settings:
+  GALAXY_LDAP_LOGGING: True
+...
+----
++
+Then run `setup.sh -e @ldapextras.yml` during {PrivateHubName} installation.
++
+[NOTE]
+====
+If it is not practical to re-run setup.sh or if debug logging is enabled for a short time, you can add a line containing `GALAXY_LDAP_LOGGING: True` manually to the `/etc/pulp/settings.py` file on {PrivateHubName}. Restart both `pulpcore-api.service` and `nginx.service` for the changes to take effect. To avoid failures due to human error, use this method only when necessary.
```
Comment (ianf77) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:162]: Should setup.sh be in code format?

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+.. To grant private automation hub access (for example, members of the `cn=pah-nosoupforyou,ou=groups,dc=example,dc=com` group): 
++
+----
+#ldapextras.yml
+---
+ldap_extra_settings:
+  AUTH_LDAP_DENY_GROUP: 'cn=pah-nosoupforyou,ou=groups,dc=example,dc=com'
+...
+----
+.. To deny private automation hub access (for example, members of the `cn=pah-nosoupforyou,ou=groups,dc=example,dc=com` group):
```
Comment (ianf77) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:136]: {PrivateHubName}

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+//.Prerequisites (Add Prereqs to 2.2 only)
+
+//* Ensure that your system is running Red Hat Ansible Automation Platform 2.2.1 or later.
+//* Ensure that you are using {PrivateHubName} 4.5.2 or later.
+
+.Procedure
+
+. Access your inventory file according to the procedure in link:hhttps://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-editing-installer-inventory-file_platform-install-scenario[Editing the Red Hat Ansible Automation Platform installer inventory file].
+
+. Use the following example as a guide to set up your Ansible Automation Platform inventory file:
```
Comment (ianf77) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:31]: {PlatformNameShort}

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+If any of these variables are missing, the Ansible Automation  installer will  not complete the installation. 
+
+
+== Setting up your inventory file variables
+
+When you configure your {PrivateHubName} with LDAP authentication, you must set the proper variables in your inventory files during the installation process.
+
+//.Prerequisites (Add Prereqs to 2.2 only)
+
+//* Ensure that your system is running Red Hat Ansible Automation Platform 2.2.1 or later.
```
Comment (ianf77) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:24]: {PlatformName}

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+//* Ensure that you are using {PrivateHubName} 4.5.2 or later.
+
+.Procedure
+
+. Access your inventory file according to the procedure in link:hhttps://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-editing-installer-inventory-file_platform-install-scenario[Editing the Red Hat Ansible Automation Platform installer inventory file].
```
Comment (ianf77) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:29]: {PlatformName}

**Review by ianf77 — COMMENTED** [review]
Some minor comments, otherwise looks good.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 1 above)
Comment (jself-sudoku) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc:15]: Done.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 2 above)
Comment (jself-sudoku) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc:162]: Yes. Updated.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 3 above)
Comment (jself-sudoku) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc:136]: Done.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 4 above)
Comment (jself-sudoku) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc:31]: Done.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 5 above)
Comment (jself-sudoku) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc:24]: Done.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 6 above)
Comment (jself-sudoku) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc:29]: Done.

**Comment by jself-sudoku** [comment]
Thanks for the review, @ianf77 . Can you squash and merge?

**Comment by jself-sudoku** [comment]
Thanks for the peer review, @ianf77. Can you squash and merge?

**Pull request merged**